scripts/test_pr_build.sh

```diff
@@ -39,7 +39,7 @@ IFS=$'\n'
 for spec in $(cat to_be_installed.${SPACK_TARGET_TYPE}.txt)
 do
     # Activate python extensions
-    if [[ "${spec}" =~ "py-" ]]
+    if [[ "${spec}" =~ "^py-" ]]
```

Comment:
This is still not optimal, because some non-python packages also extend python (like gdl).
But I think we can live with the few cases you actually need to load the package to use it in python.

Reply:
Not sure we want to activate those packages, some depend on mpi and python it is safer to have them as modules